### PR TITLE
docs: include note about canary jobs

### PR DESCRIPTION
**Pull request by justinsb**
Put in the kubetest README, as that's where I think people that would
benefit most from knowing it will be.

### CODE REVIEW

**Comment by BenTheElder** [comment]
moar kanarynetes!
/test pull-test-infra-bazel-canary

File: kubetest/README.md [hunk 1]
```diff
@@ -24,6 +24,12 @@ Note that developers frequently use `kubetest` by calling `go run hack/e2e.go`
 in the `kubernetes/kubernetes` repository. This `hack/e2e.go` program is a
 wrapper around updating `kubetest` (at most once a day) before calling it.
 
+If you're making a change to kubetest, watch the canary jobs after it has merged;
```
Comment (krzyzacy) [inline kubetest/README.md]: you still need to push a new kubekins-e2e image which will update the latest tag, probably mention here as well, either sync with HEAD then push one yourself, or ask one of us who has permission to push it.

**Comment by krzyzacy** [comment]
meh, this is better than nothing, let's get this in
/lgtm
/kind documentation

**Pull request merged**